packages/@okta/vuepress-site/docs/guides/build-sso-integration/specify-your-settings/openidconnect/settings.md

```diff
@@ -2,6 +2,17 @@
 
 * On the **General** tab, in the **Application** area, you can rename your integration and select which [grant type options](https://developer.okta.com/blog/2017/07/25/oidc-primer-part-1#whats-a-response-type) are allowed.
 
+  An OAuth 2.0 grant is the authorization granted to the client by the user. Each type of grant has a corresponding [grant flow](/docs/concepts/auth-overview/#choosing-an-oauth-2-0-flow).
+
+  The grant types permitted for OIN applications depend on the application type selected:
+
+  * Web:
+    * **Authorization code** &mdash; mandatory for Web application types
+    * Optional. **Refresh token**
+    * Optional. **Implicit (Hybrid)**
+  * SPA:
+    * Implicit &mdash; choose either **Allow ID Token with implicit grant type** or **Allow Access Token with implicit grant type**
+* The **Client acting on behalf of itself** grant type is not permitted in OIN app integrations.
```

Comment:
this would work better as a note - as a bullet, it's kind of confusing.....is it even with "Web"? Is it aligned with "On the General tab"? If so, that's extra confusing - because it's not really a step? The formatting here is hard to follow.

Reply:
Good point - I've moved it to a Note